README.md

```diff
@@ -3,3 +3,17 @@
 Materiály pro české PyLadies vytažené do samostatného repa.
 
 Brzy dostupné na naucse.python.cz.
+
+
+## License
+
+The code is licensed under the terms of the MIT license, see LICNESE.MIT file
+for full text. By contributing code to this repository, you agree to have it
+licensed with the same license.
+
+Content has it's own license specified in the appropriate matadata.
+Only [free licenses] are used. By contributing to an already licensed content,
+you agree to have it licensed with the same license. When contributing new
```

Comment:
`licensed under`

Do we want to make it clear that you can add yourself to the list of authors when making a contribution?